src/wp-includes/class-wp-block-type-registry.php

```diff
@@ -126,6 +135,15 @@ public function unregister( $name ) {
 		$unregistered_block_type = $this->registered_block_types[ $name ];
 		unset( $this->registered_block_types[ $name ] );
 
+		/**
+		 * Fires after a block type is unregistered.
+		 *
+		 * @since 6.1.0
+		 *
+		 * @param WP_Block_Type|false $block_type The unregistered block type on success, or false on failure.
```

Comment:
Same comment here about this being incorrect, it looks like it only works on successful unregistering.

Reply:
Same with the other hook, I've updated this to remove reference to `|false`